views/irmaAuthentication.php

```diff
@@ -1,13 +1,77 @@
 <?php $this->layout('base', ['pageTitle' => $this->t('Sign In')]); ?>
 <?php $this->start('content'); ?>
 <script src="<?=$this->getAssetUrl($requestRoot, 'js/irma.js'); ?>" defer></script>
+
 <script>
+
+let options = {
+      // Developer options
+      debugging: true,
+
+      // Front-end options
+      language:  'en',
+      translations: {
+        header:  'Sign the agreement with <i class="irma-web-logo">IRMA</i>',
+        loading: 'Just one second please!'
+      },
+
+      session: {
+        // IRMA server:
+        url: 'http://localhost:8088',
```

Comment:
should we make this configurable?

Reply:
Just for now, because I run the irma server locally